static/js/lib/ckeditor/plugins/LegacyShortcodes.ts

```diff
@@ -185,13 +185,8 @@ class LegacyShortcodeEditing extends Plugin {
   }
 }
 
-export default class LegacyShortcodes extends Plugin {
-  static get requires(): Plugin[] {
```

Comment:
I believe TS was treating this as the deprecated Plugin interface. https://developer.mozilla.org/en-US/docs/Web/API/Plugin